src/components/Match/renderMatch.js

```diff
@@ -56,44 +56,80 @@ function generateGraphData(match) {
   return {};
 }
 
-function generateTeamfights(match) {
+function generateTeamfights({ players, teamfights = [] }) {
   const computeTfData = (tf) => {
     const newtf = {
       ...tf,
       deaths_pos: [],
+      radiant_networth_advantage_delta: 0,
       radiant_gold_delta: 0,
+      dire_gold_delta: 0,
```

Comment:
radiant_gold_delta and dire_gold_delta are now actually the gold_delta for that fight (what the team gained/lost). the radiant_networth_advantage_delta is what was previously radiant_gold_delta. This way we have the option to display how the teams did as well as the networth advantage change.